testnet/stacks-node/src/tests/integrations.rs

```diff
@@ -32,6 +32,21 @@ use super::{
 
 use reqwest;
 
+const OTHER_CONTRACT: &'static str = "
+  (define-data-var x uint u0)
+  (define-public (f1)
+    (ok (var-get x)))
+  (define-public (f2 (val uint))
+    (ok (var-set x val)))
+";
+
+const MAIN_CONTRACT: &'static str = "
```

Comment:
Could you change the variables here to be more descriptive of what we are trying to test? (ex: `READ_ONLY_MAIN_CONTRACT`)

Reply:
will do

Reply:
addressed in 1e2ae7a